sihl-persistence/src/dune

```diff
@@ -1,4 +1,6 @@
 (library
  (name sihl_persistence)
  (public_name sihl-persistence)
- (libraries sihl-facade))
+ (libraries sihl-facade)
```

Comment:
@aronerben Please don't use ppx_fields_conv, I started to remove it in Sihl and instead use record access like `record.Some.Module.field`

Reply:
@jerben Thanks, done.